Topics/starting-a-data-project.md

```diff
@@ -0,0 +1,254 @@
+# Starting a Data Project
+
+# What you will learn 
+In this guide, you'll learn to prepare yourself to start the project. 
+
+This will consist of learning how to look for sources that can help you solve the problem, building the project structure, build good documentation, secure your code, our data and more!
+
+In this guide we'll make heavy use of an awesome best practices collection from the [PLOS | Public Library of Science](https://www.plos.org/). 
+
+Do you know the happiness when you find a _giant diamond cluster in Minecraft_? 
+
+If yes, you need to know that you've just found a fat one:
+take your time to explore the [**Ten Rules Collection**](https://collections.plos.org/ten-simple-rules).
+
+
+## Prerequisites
+All the previous Purgatorio guides.
+
+## Time to complete
```

Comment:
Are we adding this universally across all the guides, this is the first time I'm seeing it - it's an useful indicator.

Reply:
I think it's present in most of the new guides. BTW from now on we'll use it: you can find it in the template

Reply:
It's my first time I see this hence I mention, but if it is consistently present everywhere it's a good thing 👍